baryshev_code_HW7.py

```diff
@@ -0,0 +1,80 @@
+from pydrive.auth import GoogleAuth
+from pydrive.drive import GoogleDrive
+
+# Creates local webserver and auto handles authentication
+gauth = GoogleAuth()
+gauth.LocalWebserverAuth()
+
+drive = GoogleDrive(gauth)
+
+"""
+Function GetGDriveContent
+Input:
+    drive - GoogleDrive to be accessed
```

Comment:
Is 'drive' a string? What's the format for input (like is it a local directory? or is it the URL for the drive?)